internal/service/datalakepipeline/data_source_data_lake_pipeline_test.go

```diff
@@ -1,93 +0,0 @@
-package datalakepipeline_test
-
-import (
-	"fmt"
-	"os"
-	"testing"
-
-	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
-	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
-)
-
-func TestAccDataLakeDS_basic(t *testing.T) {
-	var (
-		resourceName = "mongodbatlas_data_lake_pipeline.test"
-		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName  = acc.RandomProjectName()
-		clusterName  = acc.RandomClusterName()
-		name         = acc.RandomName()
-	)
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheckBasic(t) },
-		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             checkDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: configDS(orgID, projectName, clusterName, name),
-				Check: resource.ComposeTestCheckFunc(
-					checkExists(resourceName),
-					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
```

Comment:
this was supposed to be a DS test, but was checking the resource

Reply:
not the first time I found that